packages/terra-core-docs/src/terra-dev-site/test/list/AriaDescribedBySelectableList.test.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import List, { Item } from 'terra-list/lib/index';
+
+const AriaDescribedBySelectableListTest = () => {
+  const [selected, setSelected] = useState({ selectedKey: null });
+
+  const handleSelectItem = (event, metaData) => {
+    event.preventDefault();
```

Comment:
do we need `event.preventDefault()` here..??

Reply:
Removed [cca5d7d](https://github.com/cerner/terra-core/pull/3688/commits/cca5d7d161db0999f243939667ca538e421f54d7)